src/els_completion_provider.erl

```diff
@@ -247,3 +259,13 @@ to_binary(X) when is_atom(X) ->
   atom_to_binary(X, utf8);
 to_binary(X) when is_binary(X) ->
   X.
+
+%%==============================================================================
+%% Maps definition types to completion item kinds
+%%==============================================================================
+
+-spec completion_item_kind(def_type()) -> completion_item_kind().
+completion_item_kind(define) ->
+  ?COMPLETION_ITEM_KIND_CONSTANT;
+completion_item_kind(record) ->
+  ?COMPLETION_ITEM_KIND_STRUCT.
```

Comment:
A `struct` seemed to be the best fit for records. For alternative item kinds, see the `CompletionItemKind ` namespace at https://microsoft.github.io/language-server-protocol/specifications/specification-3-14/